src/calculateRentalCost.js

```diff
@@ -4,7 +4,19 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const rentPrice = 40;
+  const shortTermDiscount = 20;
+  const longTermDiscount = 50;
+
+  if (days > 2 && days < 7) {
+    return days * rentPrice - shortTermDiscount;
+  }
+
+  if (days >= 7) {
+    return days * rentPrice - longTermDiscount;
```

Comment:
The condition checks for 7 or more days but doesn't use a constant for the value 7. Replace the literal with a named constant to improve code readability and maintainability.